From 3c14b2a80f0221ce26efb752682948b23fb0fe07 Mon Sep 17 00:00:00 2001
From: Christoph Reiter <reiter.christoph@gmail.com>
Date: Sun, 5 May 2024 17:11:29 +0200
Subject: [PATCH 26/N] Disable autodep scripts

They are Linux only, so don't include them.
The autodeps machinery itself can stay.
---
 scripts/libmakepkg/autodep/library_depends.sh.in  | 2 +-
 scripts/libmakepkg/autodep/library_provides.sh.in | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/scripts/libmakepkg/autodep/library_depends.sh.in b/scripts/libmakepkg/autodep/library_depends.sh.in
index c088baf..cb871bb 100644
--- a/scripts/libmakepkg/autodep/library_depends.sh.in
+++ b/scripts/libmakepkg/autodep/library_depends.sh.in
@@ -23,7 +23,7 @@ LIBMAKEPKG_AUTODEP_LIBRARY_DEPENDS_SH=1
 
 MAKEPKG_LIBRARY=${MAKEPKG_LIBRARY:-'@libmakepkgdir@'}
 
-autodep_functions+=('library_depends')
+# autodep_functions+=('library_depends')
 
 library_depends() {
 	if check_option "autodeps" "y"; then
diff --git a/scripts/libmakepkg/autodep/library_provides.sh.in b/scripts/libmakepkg/autodep/library_provides.sh.in
index 374cc84..ddacf43 100644
--- a/scripts/libmakepkg/autodep/library_provides.sh.in
+++ b/scripts/libmakepkg/autodep/library_provides.sh.in
@@ -23,7 +23,7 @@ LIBMAKEPKG_AUTODEP_LIBRARY_PROVIDES_SH=1
 
 MAKEPKG_LIBRARY=${MAKEPKG_LIBRARY:-'@libmakepkgdir@'}
 
-autodep_functions+=('library_provides')
+# autodep_functions+=('library_provides')
 
 library_provides() {
 	if check_option "autodeps" "y"; then
